src/client/components/SaveImageToFirebase/SaveImageToFirebase.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import firebase from './firebase'
+
+function handleImageUpload(image){
+const handleSave = () =>{
+    const bucketName = 'images';
+
+    const storageRef = firebase.storage().ref(`${bucketName}/${image}`)
+    const uploadTask = storageRef.put(image)
+    uploadTask.on(firebase.storage.TaskEvent.STATE_CHANGED,
+       () => {
+           const { downloadURL } = uploadTask.snapshot
+           return downloadURL
+       })
+      
+   }
+   return(
+    <button type="button" onClick ={handleSave}>Save </button>
+) 
```

Comment:
This formatted seems a bit off :) could you format the code?

Reply:
Yes I have pushed again after formatting